src/test/java/no/unit/nva/doi/transformer/DataciteResponseConverterTest.java

```diff
@@ -44,4 +60,40 @@ public void testToName() {
 
         Assert.assertEquals("Family, Given", name);
     }
+
+    @Test
+    public void alternativeTitlesContainLanguageTag() throws IOException, URISyntaxException {
```

Comment:
How does this fit with our test-naming pattern?

Reply:
woops :)